docs/guide/using-vue.md

```diff
@@ -200,9 +200,9 @@ Note that this might prevent certain tokens from being syntax highlighted proper
 
 ## Using CSS Pre-processors
 
-VitePress has [built-in support](https://vitejs.dev/guide/features.html#css-pre-processors) for CSS pre-processors: `.scss`, `.sass`, `.less`, `.styl` and `.stylus` files. There is no need to install Vite-specific plugins for them, but the corresponding pre-processor itself must be installed:
+VitePress has [built-in support](https://vitejs.dev/guide/features.html#css-pre-processors) for CSS pre-processors: `.scss`, `.sass`, `.less`, `.styl` and `.stylus` files. Simply install the pre-processor itself using the applicable command:
```

Comment:
again unnecessary change. please avoid just rephrasing and moving things around

Reply:
I felt the "Vite-specific plugin" point was a side-note that just added confusion when I initially read it (it led me to ask "_Am I supposed to understand the difference between a Vite-plugin and a pre-processor?_"). The revised version jumps straight to the commands necessary to get them up and running, and puts the side-note below, for those needing it.

But if you feel strongly about keeping it as is, I can revert.